doc/content/whats-new/3.33.0.md

```diff
@@ -42,7 +42,6 @@ featured:
 ### Fixed
 
 - Subscription nudge not showing correct device count in the Console.
-- Enforce default page limit on AS and NS List RPCs if a value is not provided in the request.
```

Comment:
I don't think this should be in that release changelog. Not sure what we should do about this, but it's also wrong in TTSE's changelog.

Reply:
Yes please fix it in TTSE as well.